client/src/components/Home/Home.js

```diff
@@ -160,12 +174,23 @@ class Home extends Component {
           username={this.state.username}
           profile={this.state.profile}
           contacts={this.state.contacts}
+          selected={this.state.selected}
           logout={this.logout}
           editProfile={this.editProfile}
           requestContact={this.requestContact}
           updateContact={this.updateContact}
+          selectContact={this.selectContact}
         />
-        <Chat />
+        {this.state.selected && 
+          <Chat 
+            username={this.state.username}
+            selected={
+              this.state.contacts.find(curr => curr.username === this.state.selected) 
```

Comment:
This wouldn't be efficient if a user has lots of contacts. Instead it would be better if you store the index of the contact, then this could be refactored as `selected={this.state.contacts[this.state.selected]}`

Reply:
Thanks, I fixed it in the next commit